ui_framework/doc_site/src/views/menu/menu_example.js

```diff
@@ -1,39 +1,57 @@
 import React from 'react';
 
+import { renderToHtml } from '../../services';
+
 import {
   GuideDemo,
   GuidePage,
   GuideSection,
   GuideSectionTypes,
 } from '../../components';
 
-const menuHtml = require('./menu.html');
-const menuContainedHtml = require('./menu_contained.html');
+import Menu from './menu';
+const menuSource = require('!!raw!./menu');
```

Comment:
@cjcenizal Not sure if we do ^ other places? (might be, I don't remember all the details of the UI framework code base)